lib/clouds/shared_functions.py

```diff
@@ -869,6 +872,9 @@ def is_cloud_image_uuid(self, imageid) :
         if self.get_description() == "Parallel Container Manager Cloud" :
```

Comment:
You're absolutely right. I have promised to do a re-factor on the code to remove all these. I confess that I did not finished it yet, due to the fact that will be more difficult than I initially have thought (it will involve the creation of cloud-specific ssh and security group management methods). If you agree, I would prefer to merge this "workload-centric" PR before performing an "cloud adapter-centric" refactor.